docs/EN/operate/build.md

```diff
@@ -231,42 +266,94 @@ git fetch
 git pull --recurse
 ```
 
-If you see the word error or conflict on the output, head over to [Fix Customization Conflicts](#fix-customization-conflicts).
+The first command `git fetch` brings down any updates about the state of the Trio workspace repository from GitHub.
 
-Otherwise, you are ready to build.
+The second command `git pull --recurse` does several things:
 
-### Fix Customization Conflicts
+1. Brings down any updates about the state of the submodule repositories from GitHub, e.g., `Fetching submodule XXXX` repeated for each submodule
+2. Merges any changes from GitHub into your local copy of the Trio workspace
+3. For each submodule, updates the version to match that selected in GitHub
 
-If you have customizations in the Trio code (in the workspace) that cause a conflict when you pull, you can save and restore those with the `stash` and `stash pop` lines as shown below:
+There can be errors after step 2, e.g.,
```

Comment:
"step 2" and later "step 3" might be confusing to readers, thinking the should be doing three steps, when really all three steps are done in one command by the user.

I'm not sure how to word it better, though, so you can just mark this is "resolved" if nothing better comes to mind quickly.